src/components/SelectedOption/__tests__/SelectedOption.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { render, RenderResult, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { SelectedOption } from '../SelectedOption';
+
+describe('<SelectedOption>', () => {
+    const onDeleteMock = jest.fn();
+
+    let view: RenderResult;
+
+    it('should render nothing if no children are provided', () => {
+        view = render(<SelectedOption onDelete={onDeleteMock} />);
+
+        expect(view.container).toMatchSnapshot();
+        expect(view.container).toBeEmptyDOMElement();
+    });
+
+    it('should render children if it is 0', () => {
+        view = render(<SelectedOption onDelete={onDeleteMock}>{0}</SelectedOption>);
+
+        expect(view.container).toHaveTextContent('0');
+    });
```

Comment:
but your idea of checking if 0 is there was good. What happened to it?
expect(view.container).toHaveTextContent('0');